ipywidgets/widgets/widget_description.py

```diff
@@ -21,6 +21,7 @@ class DescriptionWidget(DOMWidget, CoreWidget):
     """Widget that has a description label to the side."""
     _model_name = Unicode('DescriptionModel').tag(sync=True)
     description = Unicode('', help="Description of the control.").tag(sync=True)
+    description_html = Bool(False, help="Accept HTML in the description.").tag(sync=True)
```

Comment:
`description_html` -> `allow_html`?

Reply:
Just made a change .. to `description_allow_html` ..

Reply:
The word "allow" gives us a clue that this is a boolean. `description_html` sounds kind of like it should be an html string?